src/Mod/Draft/DraftEdit.py

```diff
@@ -26,23 +26,24 @@
                 Dmitry Chigrin, Carlo Pavan"
 __url__ = "https://www.freecadweb.org"
 
-import FreeCAD
-import Draft
+import FreeCAD as App
 import math
 
+from Draft import get3DView, getType
```

Comment:
I personally would continue using `Draft.getType`, and `Draft.get3DView`.

Fully qualified names, with their parent namespace, is better than having plain looking functions that you don't know if they are defined here or in another module. Namespaces are a good thing, according to Python's philosophy.

Reply:
+1 for reverting it. And thanks for the link, I'll have a deep read.